env/path/src/index.ts

```diff
@@ -9,18 +9,19 @@ export interface PrependDirsToPathResult {
 
 export function prependDirsToPath (prependDirs: string[], env = process.env): PrependDirsToPathResult {
   const prepend = prependDirs.join(path.delimiter)
-  if (env[PATH] != null && (env[PATH] === prepend || env[PATH]!.startsWith(`${prepend}${path.delimiter}`))) {
+  const envPath = env[PATH]
```

Comment:
Saving the `env[PATH]` lookup here allows us to remove the unsafe `!` operator below.